docs/source/user_guide.rst

```diff
@@ -152,34 +152,26 @@ In a short example, we would see coordinate conversion between Cartesian and Boy
 
 Using the functions:
 
-* :py:class:`~einsteinpy.coordinates.BoyerLindquistDifferential.to_cartesian`
-* :py:class:`~einsteinpy.coordinates.CartesianDifferential.to_bl`
+* :py:class:`~einsteinpy.coordinates.BoyerLindquist.to_cartesian`
+* :py:class:`~einsteinpy.coordinates.Cartesian.to_bl`
 
     .. code-block:: python
 
         import numpy as np
         from astropy import units as u
-        from einsteinpy.coordinates import BoyerLindquistDifferential, CartesianDifferential, Cartesian, BoyerLindquist
+        from einsteinpy.coordinates import BoyerLindquist, Cartesian
 
         a = 0.5 * u.km
 
-        pos_vec = Cartesian(.265003774 * u.km, -153.000000e-03 * u.km,  0 * u.km)
-
-        bl_pos = pos_vec.to_bl(a)
-        print(bl_pos)
-
-        cartsn_pos = bl_pos.to_cartesian(a)
-        print(cartsn_pos)
-
-        pos_vel_coord = CartesianDifferential(.265003774 * u.km, -153.000000e-03 * u.km,  0 * u.km,
+        pos_vel_coord = Cartesian(.265003774 * u.km, -153.000000e-03 * u.km,  0 * u.km,
                                   145.45557 * u.km/u.s, 251.93643748389 * u.km/u.s, 0 * u.km/u.s)
 
-        bl_coord = pos_vel_coord.bl_differential(a)
+        bl_coord = pos_vel_coord.to_bl(a)
```

Comment:
:/ `.to(BoyerLindquist)`

Reply:
These are all API breaking changes!

Reply:
@ritzvik Yes they are, should I change it to `to(classname)` or use the old methods?

Reply:
> @ritzvik Yes they are, should I change it to `to(classname)` or use the old methods?

`to(classname)` seems a good solution. You can add it but we do need to preserve the old methods also, atleast before `0.3.0`



> :/ `.to(BoyerLindquist)`

`.to(BoyerLinduist, a)` because `a` is definitely required to change other systems into BL. Maybe we can make the `.to` function like this?
`to(coordinate_class, *args)` or `to(coordinate_class, **kwargs)`